src/commons/services/utils.js

```diff
@@ -85,6 +86,23 @@
             });
         };
 
+        vm.getClusterData= function() {
```

Comment:
instead of creating new method (function) , Can we use existing one (getObjectList - function) for getting the object list. Yah i know we don't have cluster-list APi but anyway that we can handle in this way :  

1) first call the main api call with real API
2) if in case of error , than callback will goes into error callback function  
   and there we can call mock api.